eclipse-collections/src/main/java/org/eclipse/collections/impl/set/immutable/ImmutableUnifiedSet.java

```diff
@@ -43,6 +44,16 @@ public int size()
     @Override
     public boolean equals(Object other)
     {
+        if (other == this)
+        {
+            return true;
+        }
+
+        if (!(other instanceof Set))
+        {
+            return false;
+        }
+
```

Comment:
Similarly out of place

Reply:
@motlin I'm not clear why for this one: it's in a concrete class and with an implementation of `hashCode` next to it.